### PR TITLE
Update 12_how-do-i-change-my-email-address.md

### PR DESCRIPTION
**Pull request by sylee-**

### CODE REVIEW

**Comment by EGypsy** [comment]
👍

**Comment by emfet** [comment]
Looks good to me :)

**Review by Haribishop — APPROVED** [review]

File: 02_profiles/12_how-do-i-change-my-email-address.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 So you want all your FetLife email to be going to a particular email address that is different than the one you signed up on, huh? No worries!
 
-1. Click on the hamburger menu (icon with three horizontal lines) on the far right of FetLife's navigation bar.
-2. Click on “Update Settings" in the pop out menu.
-3. Click "Update" next to your email address under the Account tab.
+You can change your email address [here](https://fetlife.com/settings/account/email).
+
+Alternatively, follow these steps to update your email address.
+1. Click on the hamburger menu (the icon with three horizontal lines) at the top right-hand corner of the page.
+2. Select “Update Settings" in the dropdown menu.
```
Comment (nielsenramon) [inline 02_profiles/12_how-do-i-change-my-email-address.md:9]: First quote is different here. We need to standardize on `"` I think.

File: 02_profiles/12_how-do-i-change-my-email-address.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+3. Click "Update" next to your email address under the Account tab. Your current email address is already filled in.
```
Comment (nielsenramon) [inline 02_profiles/12_how-do-i-change-my-email-address.md:10]: `...next to your current email address...`

I would remove `Your current email address is already filled in.`

File: 02_profiles/12_how-do-i-change-my-email-address.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
...
+4. Input your new email address plus your current password.
```
Comment (nielsenramon) [inline 02_profiles/12_how-do-i-change-my-email-address.md:11]: Input your new email address **and** your current password.

File: 02_profiles/12_how-do-i-change-my-email-address.md [hunk 4]
```diff
@@ -2,6 +2,14 @@
 
 So you want all your FetLife email to be going to a particular email address that is different than the one you signed up on, huh? No worries!
```
Comment (nielsenramon) [inline 02_profiles/12_how-do-i-change-my-email-address.md:3]: Let's remove this part. Feels condescending!

File: 02_profiles/12_how-do-i-change-my-email-address.md [hunk 5]
```diff
@@ -2,6 +2,14 @@
...
 
-1. Click on the hamburger menu (icon with three horizontal lines) on the far right of FetLife's navigation bar.
-2. Click on “Update Settings" in the pop out menu.
-3. Click "Update" next to your email address under the Account tab.
+You can change your email address [here](https://fetlife.com/settings/account/email).
+
+Alternatively, follow these steps to update your email address.
+1. Click on the hamburger menu (the icon with three horizontal lines) at the top right-hand corner of the page.
```
Comment (nielsenramon) [inline 02_profiles/12_how-do-i-change-my-email-address.md:8]: `Open the FetLife Main Navigation Menu`

And FetLife Navigation Menu links to a new help document where we explain what this is and where to find it.

File: 02_profiles/12_how-do-i-change-my-email-address.md [hunk 6]
```diff
@@ -2,6 +2,14 @@
...
+2. Select “Update Settings" in the dropdown menu.
+3. Click "Update" next to your email address under the Account tab. Your current email address is already filled in.
+4. Input your new email address plus your current password.
+5. Click on the red “Update Email” button.
```
Comment (nielsenramon) [inline 02_profiles/12_how-do-i-change-my-email-address.md:12]: If we ever change this color, we will need to change it everywhere again :D

I would just state: `Submit the form`

**Review by nielsenramon — CHANGES_REQUESTED** [review]
I reviewed this one very in detail so I can go over it with @johnbaku hopefully today!

**Review by nielsenramon — APPROVED** [review]

**Comment by nielsenramon** [comment]
@fetlife/caretakers John and I reviewed and wrote this one ourselves to give you a better understanding of how we would like FAQ articles to look like in the future. Mind going over this in detail and propose other new ones 1 by 1 to me please? 🙏 

Thanks!

**Comment by BobbyGH** [comment]
Reviewed. 

I'd add one more line to each FAQ entry, something like: 

Should you still need assistance with this or something else, please drop us a line at support@fetlife.com

At least for the time being...

**Comment by nielsenramon** [comment]
@BobbyGH Let's not do that please. I don't see the point in driving more work to caretakers right now!

**Pull request merged**